src/python/env.py

```diff
@@ -149,13 +157,24 @@ def __init__(
         self.seed_game()
         self.load_game()
 
-        # initialize action space
-        self._action_set = (
-            self.ale.getLegalActionSet()
-            if full_action_space
-            else self.ale.getMinimalActionSet()
-        )
-        self.action_space = spaces.Discrete(len(self._action_set))
+        self.continuous = continuous
+        self.continuous_action_threshold = continuous_action_threshold
+        if continuous:
+            # We don't need action_set for continuous actions.
+            self._action_set = None
+            # Actions are radius, theta, and fire, where first two are the
+            # parameters of polar coordinates.
+            self.action_space = spaces.Box(
+                np.array([0, -1, 0]).astype(np.float32),
+                np.array([+1, +1, +1]).astype(np.float32),
+            )  # radius, theta, fire. First two are polar coordinates.
```

Comment:
Should radius range be `-pi` to `pi` instead? I honestly like the `-1` and `1` range, and then get the values multiplied by `pi` downstream (maybe line 275). But either way is fine I think.

```py
                np.array([0, -math.pi, 0]).astype(np.float32),
                np.array([+1, math.pi, +1]).astype(np.float32),
```

Reply:
great catch! i actually think it makes more sense to have these set here, as the environment will then expose the accurate action limits.